Managers/PresetMan.cpp

```diff
@@ -575,6 +575,47 @@ bool PresetMan::GetAllOfGroup(list<Entity *> &entityList, string group, string t
     return foundAny;
 }
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+bool PresetMan::GetAllOfGroups(std::list<Entity *> &entityList, const std::vector<std::string> &groups, const std::string &type, int whichModule) {
+	RTEAssert(!groups.empty(), "Looking for empty groups!");
+	bool foundAny = false;
+
+	if (whichModule < 0) {
+		for (DataModule *dataModule : m_pDataModules) {
+			foundAny = dataModule->GetAllOfGroups(entityList, groups, type) || foundAny;
```

Comment:
Can you do `foundAny |= dataModule->GetAllOfGroups(entityList, groups, type);`? Don't remember if that's legit in cpp or if it acts as `|` or something

Applies to the other method too

Reply:
|= is bitwise OR.

Reply:
fugg